src/common/utils.ts

```diff
@@ -26,3 +26,19 @@ export function toHumanObject(obj: unknown): unknown {
 
   return obj;
 }
+
+/**
+ * @hidden
+ */
+export function parseArgs<T>(args: string[]): T {
+  // parse args
+  const parsedArgs = args.reduce((acc, arg) => {
+    const [key, value] = arg.split('=');
+
+    return Object.assign(acc, {
```

Comment:
Why make a new object here?

The sole reference to the acc is in the reduce function. Since its not shared until reduce returns its OK to mutate

Reply:
I misremembered how assign work here BTW

What you have makes sense since its just copying over to an existing object.

I worked on a code base where the pattern was `Object.assign({}, ...stuff)` which I was thinking of